CSS/cardstyle.css

```diff
@@ -113,3 +113,32 @@
     transform: rotate(45deg);
     transition: 0.3s;
 }
+
+#arrowLeft {
```

Comment:
There is already an arrow class. Arrow left's class name should be arrow and the arrowLeft id should only contain the rotation attribute.